docs/internals/organization.txt

```diff
@@ -210,10 +209,13 @@ who demonstrate:
   This history must begin at least 18 months prior to the individual's
   candidacy for the Steering Council, and include substantive contributions in
   at least two of these bullet points:
-  - Code contributions on Django projects or major third-party packages in the Django ecosystem
+
```

Comment:
This missing blank line meant the sub-bullet was not formatted correctly (see [first bullet](https://docs.djangoproject.com/en/5.1/internals/organization/#id7))